README.md

```diff
@@ -11,6 +11,13 @@ Stable versions are available on the [Hypothesis plugin page on WordPress.org](h
 5. Click Activate
 6. Visit your WordPress Settings > Hypothesis page to configure how it works on your site
 
+## Development
+
+1. Install `php` and the `dom` and `mbstring` extensions.
```

Comment:
Is there a reference page we could link to for how to install these extensions? Are they typically included by default in the main PHP releases?

Reply:
This typically depends on the system. In Mac most of the people uses brew to install PHP and PHP extensions.

In Linux distros you can use the system package manager, but if you want the latest PHP versions you frequently have to add custom repositories.

In Windows, I have no idea 😅